make/autoconf/platform.m4

```diff
@@ -324,7 +359,9 @@ AC_DEFUN([PLATFORM_EXTRACT_TARGET_AND_BUILD],
   AC_SUBST(OPENJDK_TARGET_CPU_ARCH)
   AC_SUBST(OPENJDK_TARGET_CPU_BITS)
   AC_SUBST(OPENJDK_TARGET_CPU_ENDIAN)
+  AC_SUBST(OPENJDK_TARGET_CPU_AUTOCONF)
   AC_SUBST(OPENJDK_TARGET_LIBC)
+  AC_SUBST(OPENJDK_TARGET_ABI)
```

Comment:
I realize this has already been integrated, but I don't fully understand this. There is no corresponding change in spec.gmk.in adding OPENJDK_TARGET/BUILD_CPU_AUTOCONF and OPENJDK_TARGET/BUILD_ABI. 

So why the AC_SUBST()?

Reply:
I thought we need to declare these as `AC_SUBST` as we define the variables outside this file? I can revert these additions if you think these are incorrect.

Reply:
No, they are only needed to be able to populate the @FOO@ tags in spec.gmk.in. All the *.m4 files are concatenated into a single shell script by autoconf, so referring to variables in a different m4 file does not really matter.

I don't think it's worth a separate bug to remove them, but if you touch the area again you can get rid of them while you're at it :)

Reply:
Noted. Thanks!